### PR TITLE
Kylelaffoon image upload with folder

### PR DESCRIPTION
**Pull request by kylelaffoon**
Images for #1363

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f30bc70540/how-to/).

**Comment by stephamon** [comment]
Hey @kylelaffoon two big things: "Multi" in the article folder is spelled wrong, and you need to replace all the colons in front of the image name with a forward slash

**Comment by kylelaffoon** [comment]
@stephamon We are deleting those misspelled / mislabeled images with this request.

File: content/cloud-servers/myrackspace-two-factor-authentication.md [hunk 1]
```diff
@@ -1,175 +1,214 @@
...
-1.  Log in to the [MyRackspace portal](https://my.rackspace.com/).
+Additionally, account administrators can configure account-wide settings to specify a multi-factor authentication policy for all account users. By default, account users have the option to set up and use multi-factor authentication. Administrators can update the account-wide settings to require multi-factor authentication for all users. When this setting is enabled, users cannot access their account until they configure multi-factor authentication. Current users are logged out as soon as the account-wide requirement is applied. At the next login, users are notified of the increased security and prompted to complete the setup process.
 
-2.  In the top menu bar, click **Account** and then select **User
-    List**.
+**Notes**
 
-3.  At the top of the User List page, click **Two-Factor Login
-    Administration**.
+- You cannot configure multi-factor authentication for the Cloud Control Panel inside of MyRackspace. You must login to mycloud.rackspace.com and use these [instructions](https://support.rackspace.com/how-to/multi-factor-authentication-from-the-cloud-control-panel/) to setup your users for the Cloud CP.
```
Comment (stephamon) [inline content/cloud-servers/myrackspace-two-factor-authentication.md]: You don't need the https://support.rackspace.com in the front of the link here

**Comment by kylelaffoon** [comment]
No way to rename folder in online applicaion. Closing this PR and starting over.

**Pull request closed without merging**